src/main/java/io/confluent/idesidecar/restapi/models/graph/CcloudFlinkComputePool.java

```diff
@@ -0,0 +1,30 @@
+package io.confluent.idesidecar.restapi.models.graph;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import org.eclipse.microprofile.graphql.Name;
+
+@RegisterForReflection
+public record CcloudFlinkComputePool(
```

Comment:
nit: to be consistent with the other GraphQL-related records
```suggestion
public record CCloudFlinkComputePool(
```